sqitch.conf

```diff
@@ -1,9 +1,11 @@
 [core]
 	engine = pg
 [core "pg"]
-	db_name = genome
+	db_name = 0AAEA22466E711E5A66AA8FEA9DCD391
```

Comment:
Ooops. That wasn't intended. I amended the commit to remove the sqitch.conf changes and force pushed the new commit.